ir/instr.cpp

```diff
@@ -3577,6 +3577,66 @@ unique_ptr<Instr> Load::dup(const string &suffix) const {
   return make_unique<Load>(getType(), getName() + suffix, *ptr, align);
 }
 
+util::ConcreteVal *loadIntVal(Interpreter &interpreter,
+                              util::ConcreteValPointer *ptr,
+                              unsigned int bitwidth) {
+  assert(bitwidth >= 8);
+  assert ((bitwidth % 8) == 0); // FIXME
+  // cout << "loadIntVal\n";
+  // cout << "ptr bid= " << ptr->getBid()
+  //      << ",offset=" << ptr->getOffset()
+  //      << ",poison=" << ptr->isPoison() << "\n";
+  // cout << "getting block\n";
+  auto &cur_block = interpreter.getBlock(ptr->getBid());
+  // cur_block.print(cout);
+  auto num_bytes = bitwidth / 8;
+  if (bitwidth > (num_bytes * 8))
+    num_bytes += 1;
+  
+  auto res = new ConcreteValInt(false, llvm::APInt(bitwidth, 0));
+  bool is_ub = false;
+  for (unsigned int i = 0; i < num_bytes; ++i) {
+    auto &cur_byte = cur_block.getByte(ptr->getOffset() + i, is_ub);
+    if (is_ub){
+      interpreter.UB_flag = true;
+      return nullptr;
+    }
+
+    if (cur_byte == cur_block.default_byte || cur_byte.intValue().isPoison()) {
```

Comment:
It's impossible for `cur_byte == cur_block.default_byte` to happen, so delete that part.